scripts/prepare_sector_network.py

```diff
@@ -510,14 +510,15 @@ def add_carrier_buses(n, carrier, nodes=None):
         nodes = vars(spatial)[carrier].nodes
     location = vars(spatial)[carrier].locations
 
-    # skip if carrier already exists
-    if carrier in n.carriers.index:
+    # skip if carrier already exists (continue with coal because it can be already added by in generators)
+    if carrier in n.carriers.index and not carrier == "coal":
         return
 
     if not isinstance(nodes, pd.Index):
         nodes = pd.Index(nodes)
 
-    n.add("Carrier", carrier)
+    if not carrier == "coal":
+        n.add("Carrier", carrier)
```

Comment:
Why is `coal` already in `carriers`, but there are no coal generators & stores? Is it because the generator and store are `coal for industry`?